docs/cog_guides/alias.rst

```diff
@@ -0,0 +1,270 @@
+.. _alias:
+
+=====
+Alias
+=====
+
+This is the cog guide for the alias cog. You will
+find detailed docs about the usage and the commands.
+
+``[p]`` is considered as your prefix.
+
+.. note:: To use this cog, load it by typing this::
+
+        [p]load alias
+
+.. _alias-usage:
+
+-----
+Usage
+-----
+
+This cog allows to create shortcuts for commands.
```

Comment:
"This cog allows to" isn't grammatically correct. Maybe something more along the lines of "This cog is used to create..."